### ISSUE TITLE
events not available

### ISSUE THREAD

**Issue by travisghansen** [isindir/sops-secrets-operator#62]
Using `0.7.3` chart version I get the below (k8s 1.19)..

```
E0426 14:41:24.121724       1 event.go:263] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"ca57d051.github.com.16796f73c9c2cd26", GenerateName:"", Namespace:"operators", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"ConfigMap", Namespace:"operators", Name:"ca57d051.github.com", UID:"c17dade8-afab-432f-9f70-b2a9692b61c5", APIVersion:"v1", ResourceVersion:"191769763", FieldPath:""}, Reason:"LeaderElection", Message:"sops-operator-sops-secrets-operator-b95df6656-kxtpx_9be23d5e-a82a-48aa-98e9-2b527548b12c became leader", Source:v1.EventSource{Component:"sops-operator-sops-secrets-operator-b95df6656-kxtpx_9be23d5e-a82a-48aa-98e9-2b527548b12c", Host:""}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xc019d22506c10526, ext:21593718825, loc:(*time.Location)(0x2a15da0)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xc019d22506c10526, ext:21593718825, loc:(*time.Location)(0x2a15da0)}}, Count:1, Type:"Normal", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events is forbidden: User "system:serviceaccount:operators:sops-operator-sops-secrets-operator" cannot create resource "events" in API group "" in the namespace "operators"' (will not retry!)
```

rbac shows this:

```
- apiGroups:
  - events.k8s.io
  resources:
  - events
  verbs:
  - '*'
```

Should we add the `""` `apiGroup` in there as well?

**Comment by isindir** [comment]
@travisghansen thanks for reporting - fix is cooking, should be fixed by chart version `0.7.5` once #64 is merged

**Comment by travisghansen** [comment]
Looks good.
